src/App.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import { FC, useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+export enum Status {
```

Comment:
would be nice to have a separate file with type definitions, typedefs.tsx
So we have related types in a single place